swagger/swagger.yaml

```diff
@@ -851,21 +851,20 @@ components:
           type: string
     Scene:
       required:
-        - index
+        - id
         - end_scene_allowed
         - action_mapping
       type: object
       description: the specification for a scene in the scenario
       properties:
-        index:
-          type: integer
-          description: The order the scene appears in the scenario
-          minimum: 0
+        id:
+          type: string
+          description: The scene ID, unique throughout the scenario
         state:
           $ref: "#/components/schemas/State"
-        final_scene:
-          type: boolean
-          description: Whether this is the final scene in the scenario
+        next_scene:
+          type: string
+          description: "The ID of the default next scene in the scenario; if empty or missing, then by default this is the last scene."
```

Comment:
Reason for quotes here when they aren't in other descriptions?

Reply:
The swagger generator introduced double quotes in places, I think when there's punctuation in the string and maybe other cases.  That said, I haven't 100% made sure double quotes are used consistently throughout, but I add them when I see / think of these conditions.  I have no idea in what (if any) conditions it actually makes a difference.  Maybe it's just a swagger code generation preference.

Reply:
Good reason!